decompiler/util/default.json

```diff
@@ -220,11 +220,33 @@
       "description": "Transform while-loops to for-loops although the resulting for-loop would be empty.",
       "is_hidden_from_gui": false
     },
-    "rename_for_loop_variables": {
-      "default": true,
-      "type": "boolean",
+    "rename_for_loop_variables" : {
+      "default": ["i", "j", "k", "l", "m", "n"],
+      "type": "array",
+      "elementType": "string",
       "title": "Rename for-loop variables",
-      "description": "Rename for-loop variables to i, j, k, ...",
+      "description": "Rename for-loop variables to values from list",
+      "is_hidden_from_gui": false
+    },
+    "max_condition_complexity_for_loop_recovery": {
+      "default": 2,
```

Comment:
We should talk about the default values in the next meeting, I am not sure whether we want them this small

Reply:
Basically only two variables (or constants) would go throw. 

I mean in "normal Code" most for loops do exactly this. 
But sure will write something down until then

Reply:
Like above, value very high (100) so a user can set his/her own preferred value.